packages/components/src/components/expansion-panel/expansion-panel.jsx

```diff
@@ -0,0 +1,102 @@
+        this.setState({ isOpen: !this.state.isOpen });
+    };
+
+    onArrayItemClick = id => {
+        this.setState(state => {
+            if (state.open_id.includes(id)) {
+                const open_id = state.open_id.filter(item => {
+                    return item !== id;
+                });
+                return {
+                    ...state,
+                    open_id,
+                };
+            }
+            return {
+                ...state,
+                open_id: [...state.open_id, id],
+            };
+        });
+    };
+
+    renderArrayContent = array => {
+        return (
+            <div>
+                {array.map((item, index) => {
+                    if (Array.isArray(item.value)) {
+                        return (
+                            <div key={index} className='dc-expansion-panel__content-array'>
+                                <span className='dc-expansion-panel__content-array-item-index'>{index + 1}: </span>(
+                                {item.value.length})
+                                {this.state.open_id.includes(item.id) ? (
+                                    <Icon
+                                        className='dc-expansion-panel__content-chevron-icon'
+                                        icon='IcChevronDown'
+                                        onClick={() => this.onArrayItemClick(item.id)}
+                                    />
+                                ) : (
+                                    <Icon
+                                        className='dc-expansion-panel__content-chevron-icon'
+                                        icon='IcChevronRight'
+                                        onClick={() => this.onArrayItemClick(item.id)}
+                                    />
+                                )}
+                                {this.state.open_id.includes(item.id) ? (
+                                    <div className='dc-expansion-panel__content-array'>
+                                        {this.renderArrayContent(item.value.slice())}
+                                    </div>
+                                ) : null}
+                            </div>
+                        );
+                    }
+                    return (
+                        <div key={index} className='dc-expansion-panel__content-array'>
+                            <span className='dc-expansion-panel__content-array-item-index'>{index + 1}:</span>
+                            {item.value.toString()}
+                        </div>
+                    );
+                })}
+            </div>
+        );
+    };
+
+    render() {
+        const { message } = this.props;
+
+        return (
+            <>
+                <div className='dc-expansion-panel__header-container'>
+                    {Array.isArray(message.content) ? 'List: (' + message.header + ')' : message.header}
```

Comment:
We can't use an English string here as we don't have access to `@deriv/translations`'s, we should either only show the number or come up with a different solution.

Reply:
Have used `localize()` in `journal-notifications.js`. This will let expansion-panel to not do any specific checks, just display `message.header` instead now.